ONSdigital/eq-author-app
# 🛠 EAR 1259 - Flatten all folders into one group

### What is the context of this PR?

Squish down folders into single eQ Runner groups.

Fixes calculated summaries when referencing questions across folders.

Publisher V3 PR: https://github.com/ONSdigital/eq-publisher-v3/pull/28

### How to review

- Check questionnaires still publish okay to Runner v2
- Check skip conditions on folders still work in Runner v2
- Make a calculated summary which references questions across folders - should work when viewed in Runner v2

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
